x/upgrade/keeper/keeper.go

```diff
@@ -239,8 +239,27 @@ func (k Keeper) GetUpgradedConsensusState(ctx sdk.Context, lastHeight int64) ([]
 func (k Keeper) GetLastCompletedUpgrade(ctx sdk.Context) (string, int64) {
 	iter := sdk.KVStoreReversePrefixIterator(ctx.KVStore(k.storeKey), []byte{types.DoneByte})
 	defer iter.Close()
-	if iter.Valid() {
-		return parseDoneKey(iter.Key()), int64(binary.BigEndian.Uint64(iter.Value()))
+
+	upgrades := []struct {
```

Comment:
We can cleanup this method and have it easier to read slightly if we move this in-line struct outside of the method, it can remain private 👍